### ISSUE TITLE
Resize to parentElement option on handleUp

### ISSUE THREAD

**Issue by kas84** [nikitasnv/vue-resizable#11]
Hi! Would you be open to adding an option to resizing the element to the size of its parent when handlingUp?
Something like: 

`if (this.resizeToParentSize) this.w = this.$el.parentElement.clientWidth;`

In my case I need to calculate the size on the elements parent as a percentage (like the full width is 24h and I want it to be resizable for each hour or  half an hour).

On the parent I have a div like this:

`<div :style="getStyle(data)"></div>`

And this getStyle is calculating the width as a percentage  basing on the data.

**Comment by nikitasnv** [comment]
Do you still have a problem?

**Comment by kas84** [comment]
Yes, even though I'm not really using it, because I encountered other problems with a virtual scroll library

**Comment by nikitasnv** [comment]
I will not add additional parameters for the component if it is not needed. Have you tried setting the correct dimensions when calculating the data (width, height props)?

**Comment by kas84** [comment]
I will revisit this problem this week. I will let you know... The main problem for me was that it was not working correctly with a virtual scroll library, but maybe it was because of my resize to parentElement option.
